deploy.env

```diff
@@ -66,11 +75,12 @@ GOTRUE_ADMIN_PASSWORD=password
 # If you are using a different domain, you need to change the redirect_uri in the OAuth2 configuration
 # Make sure that this domain is accessible to the user
 # Make sure no endswith /
-API_EXTERNAL_URL=http://your-host
+# Replace with your host name instead of localhost
+API_EXTERNAL_URL=http://localhost:${GOTRUE_PORT}
```

Comment:
This will cause the login to fail, as API_EXTERNAL_URL is used in redirect URL. In the situation where this is deployed on local machine, it should be either be localhost, or localhost:(nginx port), instead of gotrue port.